frontend/src/users/enums/authorization-consts/AuthorizationConsts.ts

```diff
@@ -0,0 +1,3 @@
+export enum AuthorizationConsts {
```

Comment:
IMHO do constow lepiej robic po prostu

`
export const JWT = `JWT`
`

Te consty chyba sluza do localstorage? Jesli tak, to nigdy bym na to nie wpadl patrzac na nazwe/zawartosc pliku